QUARKUS-1415.md

```diff
@@ -0,0 +1,30 @@
+# QUARKUS-1415 Reactive REST Client improvements
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1415
+
+Test plan covers Reactive REST Client functionality provided by `quarkus-rest-client-reactive` extension.
+
+Documentation:
+https://quarkus.io/version/main/guides/resteasy-reactive
+https://quarkus.io/version/main/guides/rest-client-reactive
+
+## Related test plans
+- [QUARKUS-1075](QUARKUS-1075.md)
+
+## Scope of the testing
+- Upload and download of large(more than 2Gb) files, see https://github.com/quarkusio/quarkus/pull/22229 for details
```

Comment:
```suggestion
- Upload and download of large (more than 250MB) files, see https://github.com/quarkusio/quarkus/pull/22229 for details
```

Reply:
The original issue[1] explicitly mentions "files larger, than 2G". File of size 250MB will not really validate it.

[1] https://github.com/quarkusio/quarkus/issues/22218